packages/wonder-blocks-dropdown/generated-snapshot.test.js

```diff
@@ -227,6 +227,66 @@ describe("wonder-blocks-dropdown", () => {
         const {View} = require("@khanacademy/wonder-blocks-core");
         const {StyleSheet} = require("aphrodite");
 
+        const styles = StyleSheet.create({
+            row: {
+                flexDirection: "row",
+            },
+            dropdown: {
+                maxHeight: 200,
```

Comment:
It's too bad this doesn't show up in the snapshot.

Reply:
What do you mean, "it doesn't show up"? We have some cunning code to get aphrodite styles into the snapshot so I'm concerned there's a bug.

Reply:
It doesn't show up b/c the component doesn't show the dropdown until the user clicks on it.